docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+if [[ -z "${CERT_PATH}" || -z "${CERT_ALIAS}" ]]; then
+	echo "\"CERT_PATH\" and \"CERT_ALIAS\" are not set..skipping keystore loading"
+else
+	keytool -import -alias ${CERT_ALIAS} -keystore ${JAVA_HOME}/lib/security/cacerts -trustcacerts -file ${CERT_PATH} --storepass changeit -noprompt
```

Comment:
Are you using keytool to generate keys? Or openssl? Mind sharing commands?

Reply:
If OpenSSL is needed we have a massive pile of cert stuff that I wrote for the old HLF blockchain system where all certs were self signed. A lot of that should have made it into p8e scripts as well for the original identity keys (hopefully that is all gone by now)

Reply:
In this case there's no key involved. I used [https://self-signed.badssl.com](https://self-signed.badssl.com) for testing. So first I just attempted to reach it and it generates the Sun.PXCIS? exception. Then I used this to fetch the cert from that site. 

```bash
openssl s_client -showcerts -servername self-signed.badssl.com -connect self-signed.badssl.com:44
3 > cacert.pem
```

Once you have that you can use the keytool command above to load it into the truststore. Then running again produces a 404 NOT FOUND since the uri we have set isn't valid, but at least it passes the tls handshake.

Reply:
> If OpenSSL is needed we have a massive pile of cert stuff that I wrote for the old HLF blockchain system where all certs were self signed. A lot of that should have made it into p8e scripts as well for the original identity keys (hopefully that is all gone by now)

Some of that might be helpful for the follow up here or for Lee with another bank integration. In both those cases we need to have tomcat terminate tls.